src/platform/forms-system/test/js/components/ContactInfoReview.unit.spec.jsx

```diff
@@ -20,9 +21,9 @@ const getData = ({
   data: {
     veteran: {
       email: email ? vapProfile.email.emailAddress : null,
-      mobilePhone: mobile ? vapProfile.mobilePhone : null,
-      homePhone: home ? vapProfile.homePhone : null,
-      mailingAddress: address ? vapProfile.mailingAddress : null,
+      mobilePhone: mobile ? { ...vapProfile.mobilePhone } : null,
+      homePhone: home ? { ...vapProfile.homePhone } : null,
+      mailingAddress: address ? { ...vapProfile.mailingAddress } : null,
```

Comment:
I didn't remember while we were in the huddle, but this was necessary because we needed to clone the object to prevent data manipulation within the unit tests changing the source data. I have switched to using the `clone` function.